terraform/aws/main.tf

```diff
@@ -3,7 +3,13 @@ provider "aws" {
 }
 
 terraform {
-  required_version = ">= 0.12.20"
+  required_version = ">= 0.13.0"
```

Comment:
While this is a bump from 0.12.20, our internal automation has always used 0.13.0 as the baseline. 0.13.0 is what supports the required_providers block below.